eventfiles/class_tests/event_file_views_tests.py

```diff
@@ -37,13 +54,75 @@ def test_render_files_template(self, client, sign_in):
         "event doesnt exist",
         "user not a participant in the event",
     ])
-    def test_invalid_event_id_in_file_url(self, client, sign_in, url_path, status_code):
+    def test_invalid_event_id_in_file_url(self, client, sign_in_with_creator, url_path, status_code):
         response = client.get(url_path)
         assert response.status_code == status_code
 
-    def test_unauthorized_user_redirected_to_main_page(self, client):
+    def test_unauthorized_user_redirected_to_login_page(self, client):
         response = client.get(f'{FILE_CREATION_URL}/1')
         assert response.status_code == 302
         response = client.get(response.url)
         assert response.status_code == 200
-        assertTemplateUsed(response, 'main/index.html')
+        assertTemplateUsed(response, 'login/register_login.html')
+
+    def test_all_event_files_are_shown_and_downloadable(self, sign_in_with_user2, client):
+        char_content = self.get_char_content_from_eventfile_html(client)
+        all_file_in_event1 = EventFile.objects.get_files_by_event(EVENT_ID)
+        for file in all_file_in_event1:
+            assert f'{file}' in char_content
+            assert f'{DOWNLOAD_FILE_FROM_EVENT_URL}{file}' in char_content
+            response = client.get('/file/event_file/download/', {'event': EVENT_ID, 'file_name': f'{file}'})
+            assert response.status_code == 200
+            downloaded_file = self.get_file_name_from_response(response)
+            assert f'{file}' in downloaded_file
+
+    def test_delete_button_visible_for_file_user_created(self, sign_in_with_user2, client):
+        """ test that an ordinary user can delete only files he created """
+        char_content = self.get_char_content_from_eventfile_html(client)
+        found_file1, found_file3 = self.find_testfile1_and_testfile3(char_content)
+        assert found_file3 and not found_file1
+
+    def test_unauthorized_user_delete(self, sign_in_with_user2, client):
+        response = client.get(DELETE_EVENT_CREATOR_FILE_URL)
+        assert response.status_code == 401
+
+    def test_delete_file_created_by_user(self, sign_in_with_user2, client):
+        client.get(DELETE_USER2_FILE_URL)
+        assert not EventFile.objects.all().filter(id=FILE_CREATED_BY_USER2).exists()
```

Comment:
It is redundant to use all and then filter
filter by default uses all if not given a query before hand 😄 

```suggestion
        assert not EventFile.objects.filter(id=FILE_CREATED_BY_USER2).exists()
```

Reply:
result of copying other people code :sweat_smile

Reply:
I think you forgot to change 😄